lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -92,5 +90,6 @@ private function build($data) {
     $this->created_at = $data->timestamp;
     $this->campaign = Campaign::get($data->nid);
     $this->campaign_run = $data->run_nid;
+    $this->rbid = Reportback::get($data->rbid);
```

Comment:
ooooh this is so smart! I just started to actually understand try/catch blocks - so helpful!